app/experimenter/experiments/models.py

```diff
@@ -357,6 +357,8 @@ def targeting(self):
             sticky_clause = "is_already_enrolled"
             if is_desktop:
                 sticky_clause = "experiment.slug in activeExperiments"
+                if self.is_rollout:
+                    sticky_clause = "experiment.slug in activeRollouts"
```

Comment:
oh that's why desktop rollouts were not working 🤔 , great catch @jaredlockhart 👏